index.js

```diff
@@ -22,7 +22,7 @@ function mochaPhantomJS(options) {
     return through.obj(function (file, enc, cb) {
         var args = [
             scriptPath,
-            toURL(file.path, options.mocha),
+            escapeSpaces(toURL(file.path, options.mocha)),
```

Comment:
the escape here may not be needed since each arg should not need to be space separated. does it work without escaping here?